src/Microsoft.Sbom.Targets/Microsoft.Sbom.Targets.props

```diff
@@ -1,27 +1,21 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
-  <PropertyGroup>
-    <GenerateSbomTask_TFM Condition=" '$(TargetFramework)' == 'net6.0' ">net6.0</GenerateSbomTask_TFM>
-      <GenerateSbomTask_TFM Condition=" '$(TargetFramework)' == 'net8.0' ">net8.0</GenerateSbomTask_TFM>
-  </PropertyGroup>
-  <UsingTask TaskName="Microsoft.Sbom.Targets.GenerateSbomTask" AssemblyFile="$(MSBuildThisFileDirectory)\..\tasks\$(GenerateSbomTask_TFM)\Microsoft.Sbom.Targets.dll" />
-
   <PropertyGroup>
+        <GenerateSBOM Condition=" '$(GenerateSBOM)' == '' ">false</GenerateSBOM>
         <!-- TODO in practice these will default to existing nuget props instead of these test values.
             Need to consult with .NET SDK/ PM team for which existing props will make sense as defaults. -->
-        <SbomGenerationBuildDropPath>C:\Users\gustavoca\Repos\github\sbom-test\src\GenerateSBOMTest\GenerateSBOMTest\bin\Debug\net8.0</SbomGenerationBuildDropPath>
-        <SbomGenerationBuildComponentPath>$(MSBuildProjectDirectory)</SbomGenerationBuildComponentPath>
-        <SbomGenerationPackageSupplier>test-supplier</SbomGenerationPackageSupplier>
-        <SbomGenerationPackageName>test-name</SbomGenerationPackageName>
-        <SbomGenerationPackageVersion>1.0.0</SbomGenerationPackageVersion>
-        <SbomGenerationNamespaceBaseUri>http://spdx.org/spdxdocs/$(SbomGenerationPackageName)</SbomGenerationNamespaceBaseUri>
-        <SbomGenerationNamespaceUriUniquePart></SbomGenerationNamespaceUriUniquePart>
-        <SbomGenerationExternalDocumentReferenceListFile></SbomGenerationExternalDocumentReferenceListFile>
```

Comment:
Are we still using the `<SbomGenerationExternalDocumentReferenceListFile>` property? It might be good to still declare this in the .props file for visibility.

Reply:
I like the idea of making our props file work also as a documentation of the properties we are supporting, however I'm not sure which value we should use for SbomGenerationExternalDocumentReferenceListFile here. Since we don't want to override values set by the user, it would look like this:

```xml
<SbomGenerationExternalDocumentReferenceListFile Condition=" '$(SbomGenerationExternalDocumentReferenceListFile)' == '' "></SbomGenerationExternalDocumentReferenceListFile>
```
which basically won't do anything.

Reply:
Gotcha, that makes sense. As long as we remember to document this property somewhere, we should be good.